### PR TITLE
various

### PR DESCRIPTION
**Pull request by cmungall**

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -137302,22 +137370,28 @@ id: MONDO:0007619
...
-synonym: "fingerprints, absence of" RELATED [OMIM:136000]
-synonym: "immigration delay disease" EXACT [Orphanet:289465]
-xref: GARD:0012550 {source="MONDO:equivalentTo", source="Orphanet-shared", source="OMIM-shared"}
+synonym: "absence of fingerprints" BROAD [GARD:0012550]
+synonym: "absence of fingerprints" EXACT [DOID:0111357]
+synonym: "ADERM" EXACT ABBREVIATION [DOID:0111357, MONDO:Lexical, OMIM:136000]
+synonym: "adermatoglyphia" BROAD [DOID:0111357, MONDO:Lexical, OMIM:136000]
+synonym: "adermatoglyphia; ADERM" EXACT [OMIM:136000]
+synonym: "ADG" BROAD ABBREVIATION [GARD:0012550]
+synonym: "ADG" EXACT [DOID:0111357]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:137379]: duplicate syn

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -50456,21 +50468,26 @@ intersection_of: disease_has_location UBERON:0000473 ! testis
...
-synonym: "Sex cord-stromal tumor of testis" EXACT [DOID:4757, NCIT:C6358]
-synonym: "sex cord-stromal tumor of testis" EXACT []
-synonym: "Sex cord-stromal tumor of the testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal neoplasm of testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal neoplasm of the testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal tumor of testis" EXACT [DOID:4757, NCIT:C6358, Orphanet:363489]
+synonym: "sex cord-stromal tumor of the testis" RELATED [NCIT:C6358]
 synonym: "testicular sex cord-stromal neoplasm" EXACT [NCIT:C6358]
-synonym: "testicular Sex cord-stromal tumor" RELATED [NCIT:C6358]
+synonym: "testicular sex cord-stromal tumor" EXACT [Orphanet:363489]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:50478]: duplicate synonym

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -50456,21 +50468,26 @@ intersection_of: disease_has_location UBERON:0000473 ! testis
...
 synonym: "sex cord-stromal neoplasm" RELATED [DOID:4757]
-synonym: "Sex cord-stromal neoplasm of testis" RELATED [NCIT:C6358]
-synonym: "Sex cord-stromal neoplasm of the testis" RELATED [NCIT:C6358]
-synonym: "Sex cord-stromal tumor of testis" EXACT [DOID:4757, NCIT:C6358]
-synonym: "sex cord-stromal tumor of testis" EXACT []
-synonym: "Sex cord-stromal tumor of the testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal neoplasm of testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal neoplasm of the testis" RELATED [NCIT:C6358]
+synonym: "sex cord-stromal tumor of testis" EXACT [DOID:4757, NCIT:C6358, Orphanet:363489]
+synonym: "sex cord-stromal tumor of the testis" RELATED [NCIT:C6358]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:50476]: this is almost identical to the syn above, but this one is related and the other is exact. I think they both should be exact.

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -94818,14 +94840,16 @@ id: MONDO:0005514
...
-xref: OMIM:609549 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: OMIM:611897 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: OMIM:615972 {source="Orphanet:35612", source="EFO:0005569", source="MONDO:superClassOf", source="DOID:10629", source="ORDO:35612/btnt"}
-xref: Orphanet:35612 {source="MONDO:equivalentTo", source="DOID:10629"}
+xref: DOID:0080634 {source="MONDO:equivalentTo"}
+xref: OMIM:600165 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIM:609549 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIM:611897 {source="Orphanet:35612", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIM:615972 {source="Orphanet:35612", source="EFO:0005569", source="MONDO:superClassOf", source="DOID:10629", source="DOID:0080634", source="ORDO:35612/btnt"}
+xref: OMIMPS:600165 {source="DOID:0080634"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:94848]: I think this should be MONDO:equivalentTo

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:94848]: This OMIMPS is actually already equiv to MONDO_0021129 microphthalmia, but I think the equiv axiom should be moved to this term.
https://www.omim.org/phenotypicSeries/PS600165

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -97652,7 +97675,7 @@ synonym: "infection, Arenaviridae" RELATED [MESH:D001117]
 synonym: "infection, arenavirus" RELATED [MESH:D001117]
 synonym: "infections, Arenaviridae" RELATED [MESH:D001117]
 synonym: "infections, arenavirus" RELATED [MESH:D001117]
-xref: DOID:3944 {source="EFO:0007150", source="MONDO:obsolete"}
+xref: DOID:3944 {source="MONDO:equivalentTo", source="EFO:0007150", source="MONDO:obsolete"}
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:97678]: on line 97680

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -116390,7 +116417,7 @@ synonym: "infections, Acinetobacter" RELATED [MESH:D000151]
 synonym: "infections, Mimae" RELATED [MESH:D000151]
 synonym: "Mimae infection" RELATED [MESH:D000151]
 synonym: "Mimae infections" RELATED [MESH:D000151]
-xref: DOID:3091 {source="EFO:1000792", source="MONDO:obsolete"}
+xref: DOID:3091 {source="EFO:1000792", source="MONDO:equivalentTo", source="MONDO:obsolete"}
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:116420]: on line 116422

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -137302,22 +137370,28 @@ id: MONDO:0007619
...
-synonym: "ADERM" RELATED ABBREVIATION [MONDO:Lexical, OMIM:136000]
-synonym: "adermatoglyphia" RELATED [MONDO:Lexical, OMIM:136000]
-synonym: "adermatoglyphia; ADERM" RELATED [OMIM:136000]
-synonym: "ADG" RELATED ABBREVIATION [GARD:0012550]
-synonym: "congenital absence of fingerprints" EXACT [Orphanet:289465]
-synonym: "fingerprints, absence of" RELATED [OMIM:136000]
-synonym: "immigration delay disease" EXACT [Orphanet:289465]
-xref: GARD:0012550 {source="MONDO:equivalentTo", source="Orphanet-shared", source="OMIM-shared"}
+synonym: "absence of fingerprints" BROAD [GARD:0012550]
+synonym: "absence of fingerprints" EXACT [DOID:0111357]
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:137374]: absence of fingerprints

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:137379]: ADG

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -53490,7 +53507,7 @@ synonym: "pleura mesothelioma" EXACT [MONDO:patterns/location]
 synonym: "pleural mesothelioma" EXACT [MONDO:ambiguous, NCIT:C9351]
 synonym: "pleural mesothelioma (disease)" EXACT [MONDO:0006379]
 synonym: "pleural mesothelioma (disease), benign" NARROW [MONDO:patterns/benign]
-xref: DOID:5157 {source="MONDO:relatedTo"}
+xref: DOID:5157 {source="MONDO:relatedTo", source="MONDO:equivalentTo"}
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:53510]: Can a reference be both equivalent and related?

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -197153,7 +197436,6 @@ is_a: MONDO:0003847 {source="MESH:C564739/inferred"} ! Mendelian disease
 [Term]
 id: MONDO:0010190
 name: pontocerebellar hypoplasia type 2A
-def: "Any pontocerebellar hypoplasia type 2 in which the cause of the disease is a mutation in the TSEN54 gene." [MONDO:patterns/disease_series_by_gene]
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:197156]: Removing defs like this intended?

File: src/ontology/mondo-edit.obo [hunk 10]
```diff
@@ -205533,8 +205832,8 @@ xref: UMLS:C3669395 {source="MONDO:equivalentTo"}
 is_a: MONDO:0010556 {source="MONDO:Redundant"} ! X-linked chondrodysplasia punctata
 is_a: MONDO:0019602 {source="Orphanet:79345"} ! other inborn metabolic disease
 intersection_of: MONDO:0010556 ! X-linked chondrodysplasia punctata
-intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/719 ! ARSL
-relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/719 {source="mim2gene_medgen"} ! ARSL
+intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/719 ! ARSE
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:205835]: Seems like a serious dysfunction..

**Review by matentzn — COMMENTED** [review]
Looks ok to me. You can double check looking through my comments

File: src/ontology/mondo-edit.obo (hunk 10 above)
Comment (cmungall) [reply src/ontology/mondo-edit.obo:205835]: lol

File: src/ontology/mondo-edit.obo (hunk 7 above)
[the remark this answers is not in the document]
Comment (cmungall) [reply src/ontology/mondo-edit.obo:137374]: we'll do a broader fix of these

**Pull request merged**